ui/pages/onboarding-flow/metametrics/metametrics.js

```diff
@@ -80,26 +50,21 @@ export default function OnboardingMetametrics() {
   const onCancel = async () => {
     await dispatch(setParticipateInMetaMetrics(false));
 
-    const isInitiallyParticipatingOrNotSet =
-      participateInMetaMetrics === null || participateInMetaMetrics;
```

Comment:
presumably its no longer possible to revisit this screen once it has been hit? We used to have an issue where if the user refreshed the screen it would take them back to the beginning of onboarding and we could potentially get two opt out events for a user if this check wasn't in place. Is that no longer the case ?

Reply:
Restored!